javascripts/enemy.js

```diff
@@ -1,14 +1,40 @@
+"use strict";
+
 var FoodFight = (function(enemy) {
 
-  enemy.Player2 = function () {
+  enemy.Player2 = function (name) {
+    this.name = name;
     this.health = 50;
-    this.attack = 10;
-    this.name = "Mo";
-  }
+    this.attack = Math.floor(Math.random()*20 + 1) + 1;
+    this.allowedEnemies = [enemy.Paula, enemy.Gordon, enemy.Lovett, enemy.Guy];
+    this.generateEnemy = function() {
```

Comment:
I think we need to pull this function out of the enemy.Player2 function, so we can call it to generate a player, don't you? so lines 10 through 20 will become enemy.generateEnemy = new player2- put lines 8 and 9 inside there and return randomEnemy. then we'll call _this function_ in choosePlayer.js to generate an enemy.